docs/tutorials/getting_started/human_interaction.rst

```diff
@@ -0,0 +1,15 @@
+.. _human-interaction:
+
+Human Interaction
+=================
+
+It is possible to play interactively using the Human strategy::
+
+    >>> import axelrod as axl
+    >>> me = axl.Human(name='me')
+    >>> players = [axl.TitForTat(), me]
+    >>> match = axl.Match(players, turns=3)
+    >>> match.play() #doctest: +SKIP
+
+You will be prompted for the action to play at each turn and will be able to
+access the results of the match as described in :ref:`creating_matches`
```

Comment:
Is it worth showing some example of the prompt? It needn't be too extensive and it would not include the `>>>` so it wouldn't be picked up by the doctests. So something like (after what you have already written):

---

This will then prompt the user for inputs of `C` or `D`:

```
Starting new match
Turn 1 action [C or D] for me: C

Turn 1: me played C, opponent played C
Turn 2 action [C or D] for me: D

Turn 2: me played D, opponent played C
Turn 3 action [C or D] for me: C
[('C', 'C'), ('C', 'D'), ('D', 'C')]
```

after this the `match` object can be manipulated as described in http://axelrod.readthedocs.io/en/latest/tutorials/getting_started/match.html

---